src/components/Issues/__tests__/ConformaViolationSummaryCard.spec.tsx

```
@@ -0,0 +1,190 @@
+import { screen } from '@testing-library/react';
+import { CONFORMA_RESULT_STATUS } from '~/types/conforma';
+import type { ApplicationConformaResults, ConformaResultRow } from '~/types/conforma';
+import { APPLICATION_DETAILS_PATH } from '~/routes/paths';
```

ESLint

[error] `~/routes/paths` import should occur before import of `~/types/conforma`

(import/order)